tonic/handler.go

```diff
@@ -133,6 +139,19 @@ func Handler(h interface{}, status int, options ...func(*Route)) gin.HandlerFunc
 	return ret
 }
 
+// RegisterValidation registers a custom validation on the validator.Validate instance of the package
+// note that calling this function may instantiate the validator itself.
```

Comment:
I would indicate that this method is not thread safe, as the RegisterValidation method is not thread safe either: https://godoc.org/gopkg.in/go-playground/validator.v8#Validate.RegisterValidation

Reply:
Sure!